slam/src/slam/Util.py

```diff
@@ -0,0 +1,108 @@
+import warnings
+warnings.filterwarnings("ignore")
+warnings.filterwarnings("ignore", category=DeprecationWarning)
+warnings.filterwarnings("ignore", category=UserWarning)
+import matplotlib.pyplot as plt
+import matplotlib.animation as animation
+from matplotlib.patches import Ellipse
+import numpy as np
+import pandas as pd
+
+def wrap_to_pi(th):
+  '''Wraps its argument between [-pi, pi] element wise'''
+  return ((th + np.pi) % (2 * np.pi)) - np.pi
+
+def plot_df(history, groundtruth_path_data, landmarks_groundtruth, save_to=None, plot_avg = False, msg = ''):
+  print('msg',msg)
```

Comment:
Seems like it should probably either be removed, or only run if `msg != ''`

Reply:
It should be removed haha. Thanks!